features/inventory/cart_inventory/prevent_buying_more_products_than_available_in_a_stock.feature

```diff
@@ -0,0 +1,23 @@
+@cart_inventory
+Feature: Prevent buying more products, than available in a stock
+    In order to buy only available quantity of product's items
+    As a Visitor
+    I want to be prevented from adding product's items with quantity greater that amount of items available in the inventory
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-shirt Mononoke" priced at "$12.54"
+        And "T-shirt Mononoke" product is tracked by the inventory
+        And there are 5 items of product "T-Shirt Mononoke" available in the inventory
```

Comment:
Even if it's possible to have 5 products in the inventory but not have them tracked, it still is rather unusual and obviously not the default choice. I'm just for merging these two steps into one.